packages/wonder-blocks-dropdown/src/components/dropdown-core.js

```diff
@@ -72,18 +72,38 @@ type Labels = {|
 // values to the parent components that are instantiating this component
 // @see https://flow.org/en/docs/react/hoc/#toc-exporting-wrapped-components
 type DefaultProps = {|
-    /**
-     * An index that represents the index of the focused element when the menu
-     * is opened.
-     */
-    initialFocusedIndex?: number,
-
     /**
      * Whether this menu should be left-aligned or right-aligned with the
      * opener component. Defaults to left-aligned.
      */
     alignment: "left" | "right",
 
+    /**
+     * Whether to auto focus an option.
+     */
+    autoFocus: boolean,
+
+    /**
+     * Whether to enable the type-ahead suggestions feature.
+     *
+     * This feature allows to navigate the listbox using the keyboard.
+     * - Type a character: focus moves to the next item with a name that starts
+     *   with the typed character.
+     * - Type multiple characters in rapid succession: focus moves to the next
+     *   item with a name that starts with the string of characters typed.
+     *
+     * **NOTE:** Type-ahead is recommended for all listboxes, but there might be
+     * some cases where it's not desirable (for example when using a `TextField`
+     * as the opener element).
+     */
+    enableTypeAhead: boolean,
```

Comment:
**suggestion (non-blocking):** Could you include what the default value for this is in the docstring?